codebase/src/java/portico/org/portico/lrc/model/Mom.java

```diff
@@ -77,7 +77,9 @@
 		ObjectInstancesRegistered(32),
 		ObjectInstancesDiscovered(33),
 		TimeGrantedTime(34),
-		TimeAdvancingTime(35);
+		TimeAdvancingTime(35),
```

Comment:
Hmm I thought these changes were already in master. I'll have to have a look into that.

Reply:
As far as I can tell I didn't do any modifications to this file on the branch. I might just be missing it though. As far as I can tell changes came in during the rebase.